nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/CaptureChangeMySQL.java

```diff
@@ -0,0 +1,1001 @@
+                + "of the CDC event flow file relative to the other event flow file(s)."),
+        @WritesAttribute(attribute = "cdc.event.type", description = "A string indicating the type of CDC event that occurred, including (but not limited to) "
+                + "'begin', 'write', 'update', 'delete', 'schema_change' and 'commit'."),
+        @WritesAttribute(attribute = "mime.type", description = "The processor outputs flow file content in JSON format, and sets the mime.type attribute to "
+                + "application/json")
+})
+public class CaptureChangeMySQL extends AbstractSessionFactoryProcessor {
+
+    // Random invalid constant used as an indicator to not set the binlog position on the client (thereby using the latest available)
+    private static final int DO_NOT_SET = -1000;
+
+    // Relationships
+    public static final Relationship REL_SUCCESS = new Relationship.Builder()
+            .name("success")
+            .description("Successfully created FlowFile from SQL query result set.")
+            .build();
+
+    protected static Set<Relationship> relationships;
+
+    // Properties
+    public static final PropertyDescriptor DATABASE_NAME_PATTERN = new PropertyDescriptor.Builder()
+            .name("capture-change-mysql-db-name-pattern")
+            .displayName("Database/Schema Name Pattern")
+            .description("A regular expression (regex) for matching databases or schemas (depending on your RDBMS' terminology) against the list of CDC events. The regex must match "
+                    + "the schema name as it is stored in the database. If the property is not set, the schema name will not be used to filter the CDC events.")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor TABLE_NAME_PATTERN = new PropertyDescriptor.Builder()
+            .name("capture-change-mysql-name-pattern")
+            .displayName("Table Name Pattern")
+            .description("A regular expression (regex) for matching CDC events affecting matching tables. The regex must match the table name as it is stored in the database. "
+                    + "If the property is not set, no events will be filtered based on table name.")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
+
+    public static final PropertyDescriptor CONNECT_TIMEOUT = new PropertyDescriptor.Builder()
+            .name("capture-change-mysql-max-wait-time")
+            .displayName("Max Wait Time")
+            .description("The maximum amount of time allowed for a connection to be established, "
+                    + "zero means there is effectively no limit. Max time less than 1 second will be equal to zero.")
+            .defaultValue("0 seconds")
```

Comment:
Hmm I used to have the default as 30 seconds, not sure where that code went :( Will make the change.